test/feed-parser.test.js

```diff
@@ -25,3 +25,38 @@ test('Passing an invalid RSS feed URI should error', async () => {
 test('Passing a valid URI, but not a feed URI should error', async () => {
   await expect(feedParser('https://google.ca')).rejects.toThrow('Not a feed');
 });
+
+const assertValidFeed = (feed) => {
+  expect(Array.isArray(feed)).toBeTruthy();
+  expect(feed.length > 0).toBeTruthy();
+};
+
+
+test('Non existant feed failure case.', async () => {
+  try {
+    await feedParser('http://doesnotexists___.com');
+  } catch (err) {
+    expect(err.code).toBe('getaddrinfo ENOTFOUND doesnotexists___.com');
```

Comment:
Looks like you aren't running your own tests, since these don't pass.  Why not?

```
npm test
```

You need to compare to the error code as a string:

`expect(err.code).toBe('ENOTFOUND');`